satellite/satellitedb/migrate.go

```diff
@@ -423,535 +238,53 @@ func (db *DB) PostgresMigration() *migrate.Migration {
-					 ALTER TABLE nodes ADD free_bandwidth BIGINT NOT NULL DEFAULT -1;
-					 ALTER TABLE nodes ADD free_disk BIGINT NOT NULL DEFAULT -1;
-					 ALTER TABLE nodes ADD latency_90 BIGINT NOT NULL DEFAULT 0;
-					 ALTER TABLE nodes ADD last_contact_success TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT 'epoch';
-					 ALTER TABLE nodes ADD last_contact_failure TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT 'epoch';`,
-					// Copy data from overlay_cache_nodes to nodes
-					`UPDATE nodes
-					 SET address=overlay.address,
-					     protocol=overlay.protocol,
-						 type=overlay.node_type,
-						 free_bandwidth=overlay.free_bandwidth,
-						 free_disk=overlay.free_disk,
-						 latency_90=overlay.latency_90
-					 FROM (SELECT node_id, node_type, address, protocol, free_bandwidth, free_disk, latency_90
-						   FROM overlay_cache_nodes) AS overlay
-					 WHERE nodes.id=overlay.node_id;`,
-					// Delete the overlay cache_nodes table
-					`DROP TABLE overlay_cache_nodes CASCADE;`,
-				},
-			},
-			{
-				DB:          db.db,
-				Description: "Change bucket_id to bucket_name and project_id",
-				Version:     13,
-				Action: migrate.SQL{
-					// Modify columns: bucket_id --> bucket_name + project_id for table bucket_storage_tallies
-					`ALTER TABLE bucket_storage_tallies ADD project_id bytea;`,
-					`UPDATE bucket_storage_tallies SET project_id=SUBSTRING(bucket_id FROM 1 FOR 16);`,
-					`ALTER TABLE bucket_storage_tallies ALTER COLUMN project_id SET NOT NULL;`,
-					`ALTER TABLE bucket_storage_tallies RENAME COLUMN bucket_id TO bucket_name;`,
-					`UPDATE bucket_storage_tallies SET bucket_name=SUBSTRING(bucket_name from 18);`,
-
-					// Update the primary key for bucket_storage_tallies
-					`ALTER TABLE bucket_storage_tallies DROP CONSTRAINT bucket_storage_rollups_pkey;`,
-					`ALTER TABLE bucket_storage_tallies ADD CONSTRAINT bucket_storage_tallies_pk PRIMARY KEY (bucket_name, project_id, interval_start);`,
-
-					// Modify columns: bucket_id --> bucket_name + project_id for table bucket_bandwidth_rollups
-					`ALTER TABLE bucket_bandwidth_rollups ADD project_id bytea;`,
-					`UPDATE bucket_bandwidth_rollups SET project_id=SUBSTRING(bucket_id FROM 1 FOR 16);`,
-					`ALTER TABLE bucket_bandwidth_rollups ALTER COLUMN project_id SET NOT NULL;`,
-					`ALTER TABLE bucket_bandwidth_rollups RENAME COLUMN bucket_id TO bucket_name;`,
-					`UPDATE bucket_bandwidth_rollups SET bucket_name=SUBSTRING(bucket_name from 18);`,
-
-					// Update index for bucket_bandwidth_rollups
-					`DROP INDEX IF EXISTS bucket_id_interval_start_interval_seconds_index;`,
-					`CREATE INDEX bucket_name_project_id_interval_start_interval_seconds ON bucket_bandwidth_rollups (
-						bucket_name,
-						project_id,
-						interval_start,
-						interval_seconds
 					);`,
+					`CREATE INDEX storagenode_id_interval_start_interval_seconds ON storagenode_bandwidth_rollups ( storagenode_id, interval_start, interval_seconds );`,
```

Comment:
I'm afraid we have to name it `storagenode_id_interval_start_interval_seconds_index` so that it matches indexes made through the old migration chain.

Reply:
oops good catch, i didn't realize the index name got changed